packages/xo-server/src/index.js

```diff
@@ -40,13 +40,20 @@ import passport from 'passport'
 import { parse as parseCookies } from 'cookie'
 import { Strategy as LocalStrategy } from 'passport-local'
 
+import transportConsoleColor from '@xen-orchestra/log/transports/consoleColor'
+import { configure } from '@xen-orchestra/log/configure'
+
 // ===================================================================
 
-const debug = createLogger('xo:main')
+configure([
+  {
+    filter: process.env.DEBUG,
+
+    transport: transportConsoleColor(),
+  },
+])
```

Comment:
Let's remove this for the moment, we'll use the default behavior for now.

Reply:
Remove the whole `configure`, let's the default config apply for now.

Reply:
You're right, let's keep the configure, but add `level: 'warn'` to also display warnings/errors/fatals.